diagrams/l2_configurator_diagram.uml

```diff
@@ -0,0 +1,30 @@
+@startuml
+
+rectangle "L1" {
+  rectangle GovSimple
```

Comment:
nit: This will probably just be `Governor`

Reply:
updated